### ISSUE TITLE
Doubled single-quote escaped syntax flagged as incorrect

### ISSUE THREAD

**Issue by yevster** [Microsoft/vscode-azurearmtools#34]
Here's a line form a working `mainTemplate.json`:
```
"commandToExecute": "[concat('cd /hub*/docker-compose; sudo docker-compose down -t 60; sudo -s source /set_hub_url.sh ', reference(parameters('publicIpName')).dnsSettings.fqdn, ';  sudo docker volume rm ''dockercompose_cert-volume''; sudo docker-compose up')]"
```

The escape syntax for single quotes (`''`, i.e. two single quote characters next to each other) gets incorrectly gets red-underlined:

![image](https://user-images.githubusercontent.com/7320197/36919518-9ee98adc-1e2b-11e8-9c05-a4ea6f6d362c.png)

**Comment by StephenWeatherford** [comment]
@yevster Sorry it took so long to get to this, but the fix will be in the next release, along with some other goodies.

**Comment by StephenWeatherford** [comment]
FYI, this was fixed in the 0.6.0 release on Thursday.  Thanks again for reporting.
